test/core/iomgr/socket_utils_test.cc

```diff
@@ -60,6 +60,26 @@ static bool mutate_fd(int fd, grpc_socket_mutator* mutator) {
   return true;
 }
 
+static bool mutate_fd_2(const grpc_mutate_socket_info* info,
+                        grpc_socket_mutator* mutator) {
```

Comment:
nit: this test might serve as a slightly better usage example if it examined the `grpc_fd_usage` param passed to it